lib/build-view.js

```diff
@@ -190,9 +186,23 @@ export default class BuildView extends View {
     };
     const orientation = atom.config.get('build.panelOrientation') || 'Bottom';
     this.panel = addfn[orientation].call(atom.workspace, { item: this });
+    if (atom.config.get('build.hidePanelHeading') === false) {
+      this.resizeHeading();
+    }
+    this.fixTerminalElHeight();
     this.resizeToNearestRow();
   }
 
+  resizeHeading() {
+    const headingHeight = $('.item-views').position().top;
+    $('.build .heading').css('height', `${headingHeight}px`);
+  }
```

Comment:
I'm not sure using `.item-views' top position here works. What if there's another panel (e.g. `linter` above the build window?